packages/tradershub/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import { useHistory } from 'react-router-dom';
 import { useAuthorize, useJurisdictionStatus } from '@deriv/api';
+import { Provider } from '@deriv/library';
 import { Button, Text } from '@deriv/quill-design';
 import { TradingAccountCard } from '../../../../../components/TradingAccountCard';
 import { THooks } from '../../../../../types';
 import { MarketTypeDetails } from '../../../constants';
+import { TradeModal } from '../../../modals/TradeModal';
 import { MT5AccountIcon } from '../MT5AccountIcon';
 
 const AddedMT5AccountsList = ({ account }: { account: THooks.MT5AccountsList }) => {
     const { data: activeWallet } = useAuthorize();
```

Comment:
`activeTrading` instead of `activeWallets` since we're outside wallets package:
`const { data: activeTradingAccount } = useActiveTradingAccount();`

Reply:
@shontzu-deriv Ok, I'll change it, but this component was added not by me, I've just added TradeModal...